packages/desktop-client/src/components/FloatableSidebar.js

```diff
@@ -22,9 +22,8 @@ export function SidebarProvider({ children }) {
 }
 
 export function useSidebar() {
-  useViewportSize(); // Force re-render on window resize
-  let windowWidth = document.documentElement.clientWidth;
```

Comment:
We should be using this instead of `useViewportSize` since `useViewportSize` shrinks when you pinch-zoom in.

Reply:
@j-f1 I'm gonna need to understand this better cuz the whole ResponsiveProvider is based on `useViewportSize`. 
I know `useViewportSize` uses visual viewport...do you only see that as a problem for the sidebar logic?

Maybe ResponsiveProvider needs to expose a `clientWidth: document.documentElement.clientWidth` on its context for use cases where we need the raw pixels? So this becomes:
```
let { clientWidth } = useViewportSize();
let alwaysFloats = clientWidth < 668;
```

Reply:
@j-f1 I've exposed a `clientWidth` on the responsive context so this logic remains unchanged.

Reply:
I missed this! I don’t quite understand the benefits of the visual viewport size, since it is affected by pinch zooming. `clientWidth` feels like the thing we should be using to gate layout changes since it’s affected by both window/device size and non-pinch zooming (i.e. cmd + +/-)

Reply:
Did some reading on this and you're totally right: `clientWidth` will match what we get in our `min-width` media queries and we ought to use it for these measurements. I refactored ResponsiveProvider to do so.

Visual viewport seems like it will come in handy when we deal with on-screen keyboards on mobile. Like if we want to pin some actions right above the OSK or need the keyboard open while a combobox stretches/shrinks to fills the remainder of the screen. (These two examples are from https://react-spectrum.adobe.com/blog/building-a-combobox.html#onscreen-keyboard)